samples/caf_sensor_manager/README.rst

```diff
@@ -41,21 +41,46 @@ Single-core configuration
 =========================
 
 For the MCU with multiple cores, the default configuration uses one core to simulate the sensor and the other core to process the sensor.
-These multicore MCUs can support a single-core configuration, where the sensor is simulated and processed on a single, selected core.
+The multicore MCUs can also support a single-core configuration, where the sensor is simulated and processed on a single, selected core.
 The configuration is placed in the :file:`boards/<board>_singlecore.conf` file.
 
-To use this configuration, specify the ``-DOVERLAY_CONFIG=boards/<board>_singlecore.conf`` parameter along with the build command when building the sample:
+To use this configuration, run the following command:
 
-.. code-block:: console
+   .. tabs::
 
-   west build -b nrf5340dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG=boards/nrf5340dk_nrf5340_cpuapp_nrf5340_singlecore.conf
+      .. group-tab:: nRF5340 DK
+
+         .. code-block:: console
+
+            west build -b nrf5340dk_nrf5340_cpuapp -- -DOVERLAY_CONFIG=boards/nrf5340dk_nrf5340_cpuapp_nrf5340_singlecore.conf
 
 Building and running
 ********************
 
+For the MCU with multiple cores, the default configuration ensures that all the required cores are built.
+You can build and flash all the required images by completing the following steps for all the required cores.
+
 .. |sample path| replace:: :file:`samples/caf_sensor_manager`
 
-.. include:: /includes/build_and_run.txt
+   .. group-tab:: nRF5340 DK
+
+      .. include:: /includes/build_and_run.txt
+
+      Complete the following steps to program the sample:
```

Comment:
The doc build error is because the tab was not started with ".. tabs::" tag ( like in line no: 49). 

We could remove tabs from the documentation because now details of only one DK is mentioned unlike in the earlier case, where multiple DK details were mentioned.

```suggestion
.. include:: /includes/build_and_run.txt

Complete the following steps to program the sample:
```